docs/developers/data-model.md

```diff
@@ -0,0 +1,79 @@
+## Table of Contents
+
+  - [Flow of data in OpenTrials](#flow-of-data-in-opentrials)
+  - [API database](#api-database)
+  - [Warehouse database](#warehouse-database)
+
+## Flow of data in OpenTrials
+
+1. Data about trials, publications etc. is gathered from different registries in 
+[Collectors](https://github.com/opentrials/collectors).
+2. Most registries have a corresponding table in our `warehouse` database that
+follows their data structure. The data gathered from each registry is inserted
+into that corresponding table after very little processing (e.g. type conversion, splitting).
+3. Data from the `warehouse` is normalized in [Processors](https://github.com/opentrials/processors)
+to comply with the structure
+of our [API](https://github.com/opentrials/api) `database`. This process involves:
```

Comment:
Unfortunately it is ever since we [started referring to our API database as `database`](https://github.com/opentrials/processors/blob/master/.env.example#L5).